src/App.js

```diff
@@ -28,7 +29,8 @@ function App() {
     <div className="App">
       {location.pathname === "/home" ||
       location.pathname === "/archived" ||
-      location.pathname === "/label" ? (
+      location.pathname === "/label" ||
+      location.pathname === "/trash" ? (
         <Header />
```

Comment:
Instead of writing so many logical OR statements you can simply do something like this ``` ["/home","/archived","/label","/trash"].includes(location.pathname) && <Header/> ```

Reply:
Nice suggestion will implement this.